tools/serverpod_cli/lib/src/analyzer/models/validation/restrictions.dart

```diff
@@ -211,6 +211,69 @@ class Restrictions {
     return [];
   }
 
+  List<SourceSpanSeverityException> validateExtendsClassKey(
+    String parentNodeName,
+    dynamic extendsClass,
+    SourceSpan? span,
+  ) {
+    if (extendsClass is! String) {
+      return [
+        SourceSpanSeverityException(
+          'The "extends" key must be a String.',
```

Comment:
Suggestion(NIT): Refer to the `Keywords.extendsClass` keyword.

If we decide to change the keyword this change will then propagate to all the error messages :)
Same for the other messages.